### ISSUE TITLE
theme not recognized

### ISSUE THREAD

**Issue by Maxiride** [squidfunk/mkdocs-material#454]
## Description
After running `mkdocs serve` in the shell I get:
`ERROR   -  Config value: 'theme'. Error: Unrecognised theme. `

### Expected behavior

The docs with the theme be show locally.

### Actual behavior

No docs loaded

### Steps to reproduce the bug

1. `pip install mkdocs-material` :heavy_check_mark: 
2. Add `theme: 'material' `
3. Run `mkdocs serve`

### Package versions

* Python: Python 2.7.12
* MkDocs: version 0.16.3
* Material: Version: 1.10.0

### Project configuration

``` yaml
site_name: PHP Telegram Bot Documentation

pages:
    - Home: index.md
    - About: about.md
    - Code Playground: test.md

theme: 'material'

markdown_extensions:
    - smarty
    - abbr #https://pythonhosted.org/Markdown/extensions/abbreviations.html
    - def_list #https://pythonhosted.org/Markdown/extensions/definition_lists.html
    - footnotes #https://pythonhosted.org/Markdown/extensions/footnotes.html
    # - pymdownx.emoji:
    #   emoji_generator: !!python/name:pymdownx.emoji.to_svg

```

### System information

* OS: KDE Neon
* Browser: Chromium

**Comment by Maxiride** [comment]
After reading the extensive guiode [here](https://squidfunk.github.io/mkdocs-material/getting-started/#troubleshooting) I found out why it is not working. I've installed mkdocs via apt and the them via pip.
:tada: 

Thanks for this beautiful theme :+1:

**Comment by abannachGrafana** [comment]
I ran into a similar problem. I had installed mkdocs via pip "globally" and then tried setting up a virtual environment and installing it there via `pip install mkdocs-material` What I didn't realize is that the virtual environment defaulted to the globally installed mkdocs instead of the locally installed version. So basically `mkdocs-material` was not visible to the binary that was being executed. Uninstalling from the global environment resolved the issue for me.
